### ISSUE TITLE
Create a m3u for Puffer

### ISSUE THREAD

**Issue by disableddabel** [dtankdempse/free-iptv-channels#24]
https://puffer.stanford.edu/player/

https://github.com/StanfordSNR/puffer

The links may help you!

**Comment by pucknbass** [comment]
i dont think you are understanding iptv and the project properly ... this need a username and password so it tokenized
